test/e2e/focusable.cy.js

```diff
@@ -25,160 +25,169 @@ describe('focusable', () => {
   });
 
   describe('example fixtures', () => {
-    [undefined, 'full', 'legacy-full'].forEach((displayCheck) => {
```

Comment:
all that change in tests cases is the addition of 'full-native' as a parameter. The formatter exploded the diff a bit 😁